python/src/aiconfig/editor/server/server.py

```diff
@@ -63,14 +67,21 @@ class ServerState:
 @dataclass(frozen=True)
 class HttpPostResponse:
     message: str
-    output: str | None = None
+    aiconfig: AIConfigRuntime | None
     code: int = 200
```

Comment:
We'll be able to customize this response a bit for other endpoints, right? Like list_models should just return a list of supported/registered model names

Reply:
Yes. You tell me the ideal API, i'll do it. But yes I have list_models like this:

```
+@app.route("/api/list_models", methods=["GET"])
+def list_models() -> FlaskResponse:
+    out: list[str] = ModelParserRegistry.parser_ids()  # type: ignore
+    json_obj = core_utils.JSONObject({"data": core_utils.JSONList(out)})
+    return FlaskResponse((json_obj, 200))
```